### PR TITLE
Styling/jkw/home page

### LINKED ISSUE #21: Restyle Home Page
On the home page, different book lists should be broken into sections that take up horizontal space:
     - [ ] Add flex styling to container (`row`)
     - [ ] Re-style book cards to be smaller
     - [ ] Horizontal scroll

### PR DESCRIPTION
**Pull request by kip-west**
#### What's this PR do?
- Reformats the Home Page (via `Books` and `Book` components) to integrate a horizontal, scrollable container.

Type of change made:

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Styling- no new features

#### Where should the reviewer start?
- Check out these files in tandem

#### Why is this change required? What problem does it solve?
- Since our UI seems to be headed in the direction of many scrollable lists, this sort of styling will allow us to stack them on the same page.

#### Were there any challenges that arose while implementing this feature? If so, how were they addressed?
- I'm not entirely happy with the styling on the Cards!! There are some spacing issues that I'd like to get worked out, regarding the book image

#### What are the relevant tickets?
- Close #21 

#### Screenshots (if appropriate)
- jkw-restyle-homePage

### CODE REVIEW

File: src/Components/Books/Books.css [hunk 1]
```diff
@@ -1,12 +1,23 @@
...
+
 .books-container {
-  display: grid;
-  grid-gap: 35px;
-  width: 100%;
-  grid-template-columns: repeat(auto-fit, minmax(250px, 1fr));
-  background: seashell;
+  display: flex;
+  flex-direction: row;
+  overflow: scroll;
```
Comment (NicoleLatifi) [inline src/Components/Books/Books.css:11]: This is great! I love the horizontal scroll.

**Review by NicoleLatifi — APPROVED** [review]

File: src/Components/Book/Book.js [hunk 2]
```diff
@@ -6,16 +6,16 @@ import { Link } from 'react-router-dom';
 
 const Book = ({ book, addBook }) => {
   return (
-    <section className="book-card-section">
-        <h1>{book.title}</h1>
-        <h3>{book.author}</h3>
-        <h3>Ranking: {book.rank}</h3>
+    <section className="book-card">
+        <div className="card-header">
+          <h3 className="card-text">#{book.rank}</h3>
```
Comment (NakiNorton) [inline src/Components/Book/Book.js:11]: The book rank looks great like this! Awesome idea.

**Review by NakiNorton — APPROVED** [review]
This is looking fantastic! Excited to see what it'll look like when we get multiple genres displaying.

**Comment by amykarnaze** [comment]
Horizontal scroll

**Pull request merged**